tests/providers/databricks/hooks/test_databricks.py

```diff
@@ -178,11 +179,18 @@ def uninstall_endpoint(host):
 
 def list_jobs_endpoint(host):
     """
-    Utility function to generate the list jobs endpoint giver the host
+    Utility function to generate the list jobs endpoint given the host
     """
     return f'https://{host}/api/2.1/jobs/list'
 
 
+def list_zones_endpoint(host):
+    """
+    Utility function to generate the list zones endpoint given the host
```

Comment:
if docstring single line then I prefer to keep the entire string in just one line like below, however, In this repo I can see at some place it in a single like at someplace not.
```
"""Utility function to generate the list zones endpoint given the host"""
```

Reply:
addressed in [b97106d](https://github.com/apache/airflow/pull/24617/commits/b97106d5380de3eee713dc63fe240c20bf6f2b5c)